src/index.ts

```diff
@@ -101,19 +102,72 @@ export class EppoJSClient extends EppoClient {
   // Ensure that the client is instantiated during class loading.
   // Use an empty memory-only configuration store until the `init` method is called,
   // to avoid serving stale data to the user.
+  /**
```

Comment:
I am not sure why, but github does not seem to be showing the diff against typo/no-singleton / #170